### PR TITLE
[AIRFLOW-4001] Update docs about how to run tests

### PR DESCRIPTION
**Pull request by milton0825**
Make sure you have checked _all_ steps below.

### Jira

- [x] My PR addresses the following [Airflow Jira](https://issues.apache.org/jira/browse/AIRFLOW/) issues and references them in the PR title. For example, "\[AIRFLOW-XXX\] My Airflow PR"
  - https://issues.apache.org/jira/browse/AIRFLOW-4001
  - In case you are fixing a typo in the documentation you can prepend your commit with \[AIRFLOW-XXX\], code changes always need a Jira issue.
  - In case you are proposing a fundamental code change, you need to create an Airflow Improvement Proposal([AIP](https://cwiki.apache.org/confluence/display/AIRFLOW/Airflow+Improvements+Proposals)).

### Description

- [X] Here are some details about my PR, including screenshots of any UI changes:
- Updated docs about how to run tests
- Update `run-ci.sh` to support pass `nose_args`
### Tests

- [X] My PR adds the following unit tests __OR__ does not need testing for this extremely good reason:
Update docs and ci script.

### Commits

- [X] My commits all reference Jira issues in their subject lines, and I have squashed multiple commits if they address the same issue. In addition, my commits follow the guidelines from "[How to write a good git commit message](http://chris.beams.io/posts/git-commit/)":
  1. Subject is separated from body by a blank line
  1. Subject is limited to 50 characters (not including Jira issue reference)
  1. Subject does not end with a period
  1. Subject uses the imperative mood ("add", not "adding")
  1. Body wraps at 72 characters
  1. Body explains "what" and "why", not "how"

### Documentation

- [ ] In case of new functionality, my PR adds documentation that describes how to use it.
  - When adding new operators/hooks/sensors, the autoclass documentation generation needs to be added.
  - All the public functions and the classes in the PR contain docstrings that explain what it does

### Code Quality

- [ ] Passes `flake8`

### CODE REVIEW

**Comment by milton0825** [comment]
@feng-tao

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -160,18 +160,17 @@ There are three ways to setup an Apache Airflow development environment.
...
-  tox -e py35-backend_mysql
+  export TOX_ENV=py27-backend_mysql-env_docker
+  /app/scripts/ci/run-ci.sh
   ```
 
   If you wish to run individual tests inside of Docker environment you can do as follows:
 
   ```bash
-    # From the container (with your desired environment) with druid hook
-    tox -e py35-backend_mysql -- tests/hooks/test_druid_hook.py
```
Comment (feng-tao) [inline CONTRIBUTING.md:173]: does this line doesn't work?

Comment (milton0825) [reply CONTRIBUTING.md:173]: This step and the step above does not work anymore.

Comment (milton0825) [reply CONTRIBUTING.md:173]: There is some additional setup in the `run-ci.sh` not captured here.

Comment (feng-tao) [reply CONTRIBUTING.md:173]: thanks.

**Pull request merged**